apps/account-api/src/services/accounts.service.ts

```diff
@@ -108,4 +114,21 @@ export class AccountsService {
     }
     throw new Error('Invalid Sign In With Frequency Request');
   }
+
+  getRetireMsaPayload(accountId: string): Promise<{ signerPayload: SignerPayloadRaw; encodedPayload: string }> {
+    return this.blockchainService.createRetireMsaPayload(accountId);
+  }
```

Comment:
Question for discussion:

Should we proactively check some pre-conditions here before generating the extrinsic? Specifically:
- Check that `accountId` maps to a valid MSA
- Check that the MSA is eligible to be retired (ie, no active delegations)

These checks are done by the extrinsic itself, but I'm wondering if it's beneficial to do them here as well to avoid submitting an extrinsic that could ultimately fail? Granted, it's free, and we have a signature replay registry in the node to mitigate a replay attack, I believe... but I'm still thinking it's not a bad idea here?

Thoughts?